config/jobs/image-pushing/k8s-staging-seccomp-operator.yaml

```diff
@@ -14,7 +14,7 @@ postsubmits:
               - /run.sh
             args:
               - --project=k8s-infra-staging-seccomp-operator
-              - --scratch-bucket=gs://k8s-infra-staging-seccomp-operator
+              - --scratch-bucket=gs://k8s-infra-staging-seccomp-operator-gcb
```

Comment:
`gs://k8s-staging-seccomp-operator-gcb`

need to drop `-infra`

Reply:
Ah nice! Will update

Reply:
@dims just checking, the fact that `infra` is used here won't be a problem? https://github.com/kubernetes/k8s.io/pull/1017#issuecomment-660949271

Reply:
can you please peek at the script output to verify the name of the acl vs name of bucket etc. https://github.com/kubernetes/k8s.io/pull/1045#issuecomment-668820036

(i can check after the call i am in)